src/get-started/setup-an-ide.md

```diff
@@ -107,3 +109,30 @@ The third config is `unit test`, which enables you can run/debug unit tests. You
 ![unit test output](../img/unit-test-output.png)
 
 If you encounter any problems during your journey, do not hesitate to reach out on the [TiDB Internals forum](https://internals.tidb.io/).
+
+## Visual Studio Code
+
+VS Code is a generic IDE that has good extensions for working with Go and TiDB.
+
+![VS Code with TiDE](../img/vscode_tide.png)
+
+### Prerequisites
+
+* `go`: TiDB is a Go project thus its building requires a working `go` environment. See the previous [Install Golang](install-golang.md) section to prepare the environment.
+* TiDB source code: See the previous [Get the code, build and run](build-tidb-from-source.md) section to get the source code.
+
+### Download VS Code
+
+Download GoLand from [here](https://code.visualstudio.com/Download) and install it.
+
+Now install these extensions:
+
+* [Go](https://marketplace.visualstudio.com/items?itemName=golang.Go)
+* [TiDE](https://marketplace.visualstudio.com/items?itemName=dragonly.ticode)
+* [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=github.vscode-pull-request-github)
```

Comment:
```suggestion
```
I think this extension is irrelevant?

Reply:
I think having this extension makes sense as GitHub is core the the contribution workflow. We could mark it as optional, but then we should probably label TiDE as optional as well.